static/js/util/api_test.js

```diff
@@ -0,0 +1,35 @@
+import assert from 'assert';
+import fetchMock from 'fetch-mock/src/server';
+import { getCourseList } from './api';
+
+describe('common api functions', function () {
+  this.timeout(5000);
+
+  it('gets a list of courses', done => {
+    fetchMock.mock('/api/v0/courses/', () => {
```

Comment:
Can you also assert that the list of courses coming from the API is being returned in the `getCourseList` promise? For example: https://github.com/mitodl/teachersportal/blob/master/static/js/util/api_test.js#L27